src/cms/page-builder-media.md

```diff
@@ -3,7 +3,7 @@ ee_only: true
 title: Page Builder - Media
 ---
 
-Use the _Media_ section of the Page Builder panel to add images, video, banners, sliders, and Google Maps to any layout container on the stage. When a media content type is dragged from the panel to the stage, a container appears with a toolbox of options that are specific to the content type.
+Use the _Media_ section of the Page Builder panel to add images, video, banners, sliders, and Google Maps to any layout container on the [Page Builder stage]({% link cms/page-builder-workspace.md %}#stage). When a media content type is dragged from the panel to the stage, a container appears with a toolbox of options that are specific to the content type.
```

Comment:
not sure if there is a difference, but all other sections start with "use the XXXX content type" instead of section.

Reply:
This is a section and not a content type.